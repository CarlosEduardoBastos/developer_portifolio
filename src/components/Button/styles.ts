import styled, { css } from 'styled-components';

export const Container = styled.button`
  ${({ theme }) => css`
    background-color: ${theme.colors.secondaryColor};
    border: none;
    padding: 1rem 2.5rem;
    font-size: 1rem;
    cursor: pointer;
    transition: .3s ease-in-out;

    &:hover{
      border: 1px solid ${theme.colors.secondaryColor};
      background-color: transparent;
    }
  `}
`;
