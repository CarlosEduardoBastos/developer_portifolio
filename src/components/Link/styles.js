import styled, { css } from 'styled-components';

export const Container = styled.a`
  ${({ theme }) => css`
    font-size: ${theme.fonts.sizes.medium};
    color: ${theme.colors.colorLink};
  `}
`;
