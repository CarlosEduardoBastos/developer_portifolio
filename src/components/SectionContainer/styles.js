import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css`
    max-width: 85.375rem;
    margin: 0 auto;
    padding: 0 4rem;
  `}
`;
