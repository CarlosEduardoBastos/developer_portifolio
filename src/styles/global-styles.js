import { createGlobalStyle, css } from 'styled-components';

export const GlobalStyles = createGlobalStyle`
  ${({ theme }) => css`
    * {
      margin: 0;
      padding: 0;
      box-sizing: border-box;
    }
    body {
      font-family: ${theme.fonts.family.default};
    }
    a {
      text-decoration: none;
    }
  `}
`;
