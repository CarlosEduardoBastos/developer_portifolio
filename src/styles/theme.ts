export const theme = {
  colors: {
    primaryColor: '#0a0a0a',
    secondaryColor: '#fbe850',
    colorSection: '#2e3038',
    colorLink: '#adadad',
    colorText: '#9d9d9f',
    white: '#FFFFFF',
  },
  fonts: {
    family: {
      default: "'Inter', sans-serif",
      secondary: "'Prata', serif"
    },
    sizes: {
      small: '1rem',
      xsmall: '1.375rem',
      medium: '1.5rem',
      xmedium: '1.5625',
      large: '2rem',
      xlarge: '2.375rem',
      xxlarge: '3rem',
      huge: '3.1875rem',
    },
  },
  media: {
    lteMedium: '(max-width: 1039px)',
  },
  opacity: {
    default: 0.7,
  },
};
