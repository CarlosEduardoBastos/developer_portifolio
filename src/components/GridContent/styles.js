import styled, { css } from 'styled-components';
import {Container as TextComponent} from '../TextComponent/styles'

export const Container = styled.div`
  ${({ theme }) => css`
    display: flex;
    justify-content: space-between;
    flex-wrap: wrap;
    align-items: center;

    ${TextComponent}{
      margin: 10px 0;
    }
  `}
`;

export const Content = styled.div`
  ${({ theme }) => css`
  max-width: 40%;
  `}
`;
export const ContentText = styled.div`
  ${({ theme }) => css`
  max-width: 50%;
  `}
`;
