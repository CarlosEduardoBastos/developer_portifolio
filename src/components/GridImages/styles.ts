import styled, { css } from 'styled-components';
import {Container as Heading} from "../Heading/styles"

export const Container = styled.div`
  ${({ theme }) => css`
    display: flex;
    align-items: center;
    justify-content: space-between;
    flex-wrap: wrap;

    ${Heading} {
      margin: 10px 0;
      font-weight: 200;
    }
  `}
`;

export const Spense = styled.div`
  ${({ theme }) => css`
    width: 550px;
  `}
`;
export const SpenseImageContent = styled.div`
  ${({ theme }) => css`
    width: 100%;
  `}
`;

export const Image = styled.img`
  ${({ theme }) => css`
    max-width: 100%;
  `}
`;
export const SpenseText = styled.div`
  ${({ theme }) => css`
    max-width: 540px;
  `}
`;
export const YelpCamp = styled.div`
  ${({ theme }) => css`
    width: 550px;
    margin: 5px 0;
  `}
`;

export const YelpCampImageContent = styled.div`
  ${({ theme }) => css`
    width: 100%;
  `}
`;

export const YelpCampText = styled.div`
  ${({ theme }) => css`
    max-width: 540px;
  `}
`;