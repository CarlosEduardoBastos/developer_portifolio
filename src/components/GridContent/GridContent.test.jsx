
import { GridContent } from '.';
import { renderTheme } from '../../styles/render-theme';

describe('<GridContent />', () => {
  it('should match snapshot', () => {
    const {container} = renderTheme(<GridContent />);
    expect(container.firstChild).toMatchSnapshot()
  });
});
