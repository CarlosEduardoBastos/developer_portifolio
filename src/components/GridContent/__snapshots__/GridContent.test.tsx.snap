// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridContent /> should match snapshot 1`] = `
<div
  class="sc-gswNZR giOKtt"
>
  <div
    class="sc-hLBbgP klxQqr"
  >
    <div
      class="sc-eDvSVe isSCPF"
    >
      <h2
        class="sc-bcXHqe kPGDVh"
      >
        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been
      </h2>
      <p
        class="sc-dkrFOg fBdHXp"
      >
        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has
      </p>
      <p
        class="sc-dkrFOg fBdHXp"
      >
        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has
      </p>
    </div>
    <div
      class="sc-jSUZER QlcaK"
    >
      <p
        class="sc-dkrFOg fBdHXp"
      >
        It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.
      </p>
      <p
        class="sc-dkrFOg fBdHXp"
      >
        of letters, as opposed to using 'Content here, content here'
      </p>
      <p
        class="sc-dkrFOg fBdHXp"
      >
        making it look like Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by
      </p>
      <p
        class="sc-dkrFOg fBdHXp"
      >
        Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin
      </p>
    </div>
  </div>
</div>
`;
